### DISCUSSION TITLE
Any reason 6.1 has not been backported to Kirkstone?

### DISCUSSION THREAD

**Discussion by jaskij** [Freescale/meta-freescale#1468] [Q&A] [answered]
As it is, Kirkstone, the LTS release, is using a kernel which is [no longer supported upstream](https://lore.kernel.org/lkml/1673522302104114@kroah.com/).

I do not think this is a major issue at the moment, but wanted to bring attention to it.

Of note, as I'm upgrading the kernel anyway, I will do some testing with 6.1.4 (current master in this repo). Or should I go directly to a newer version?

My BSP is customized, such as using Mesa 22.3.7, but it will hopefully be at least an indicator if this works.

**Comment by otavio** [comment, marked as answer]
I have prepared a branch in linux-fslc for it. Please prepare a PR for it, against master and then we do the backport.
